tests/meson.build

```diff
@@ -155,6 +155,11 @@ if get_option('enable-tensorflow-lite') and get_option('enable-edgetpu')
   subdir('nnstreamer_filter_edgetpu')
 endif
 
+if get_option('enable-movidius-ncsdk2')
```

Comment:
add `get_option('enable-movidius-ncsdk2')` at line 136 to install the corresponding model files used in the test?

Reply:
Right. Thanks.